jdaviz/tests/test_subsets.py

```diff
@@ -308,3 +310,82 @@ def test_disjoint_spectral_subset(cubeviz_helper, spectral_cube_wcs):
     subset_plugin.subset_selected = "Create New"
     subset_plugin.subset_selected = "Subset 1"
     assert subset_plugin._get_value_from_subset_definition(0, "Lower bound", "value") == 30
+
+
+def test_composite_region_from_subset_3d(cubeviz_helper):
+    data = Data(flux=np.ones((128, 128, 256)), label='Test 3D Flux')
```

Comment:
Do you really need 256 slices for the tests, can we do less like 10? (Assuming the last axis here is the spectral, I can never be sure with specutils swapping things around.)